packages/compartment-mapper/src/compartment-map.js

```diff
@@ -467,6 +467,7 @@ const assertEntry = (allegedEntry, url) => {
  * @param {string} [url]
  * @returns {asserts compartmentMap is import('./types.js').CompartmentMapDescriptor}
  */
+
```

Comment:
This newline seems out-of-place. I’m sure it’s omissible nearly everywhere and I’m not sure why this one’s different.

Reply:
Without both the `return undefined;` I removed at line 491, and this newline, I get
```
$ yarn lint
yarn run v1.22.0
$ yarn lint:types && yarn lint:js
$ tsc -p jsconfig.json
$ eslint .

/Users/markmiller/src/ongithub/endojs/endo/packages/compartment-mapper/src/compartment-map.js
  465:1  warning  JSDoc @returns declaration present but return expression not available in function  jsdoc/require-returns-check

✖ 1 problem (0 errors, 1 warning)
```

See the similar example at https://github.com/Agoric/agoric-sdk/pull/5597

and speculation at https://github.com/Agoric/agoric-sdk/pull/5597#issuecomment-1155936997

> looking at the shell trace of the failure I record above, it looks like typescript itself doesn't fail mysteriously, but eslint does, based on eslint also trying to interpret jsdoc type declarations.